From 1ef33ab241881264a82967cc38ce18979b480b62 Mon Sep 17 00:00:00 2001
From: Tamas Ban <tamas.ban@arm.com>
Date: Wed, 21 Feb 2024 12:25:02 +0100
Subject: [PATCH 850/954] refactor(psa): rename all 'rss' files to 'rse'

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 952a86c8d36891b0520c4f1f183495203d9dfe85 refactor(psa): rename all 'rss' files to 'rse']
Signed-off-by: Tamas Ban <tamas.ban@arm.com>
Change-Id: I379c471c541dda25d8ee9087fcf67e05b4204474
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/443842
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Tested-by: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 include/lib/psa/{rss_crypto_defs.h => rse_crypto_defs.h}   | 0
 include/lib/psa/{rss_platform_api.h => rse_platform_api.h} | 0
 lib/psa/{rss_platform.c => rse_platform.c}                 | 0
 3 files changed, 0 insertions(+), 0 deletions(-)
 rename include/lib/psa/{rss_crypto_defs.h => rse_crypto_defs.h} (100%)
 rename include/lib/psa/{rss_platform_api.h => rse_platform_api.h} (100%)
 rename lib/psa/{rss_platform.c => rse_platform.c} (100%)

diff --git a/include/lib/psa/rss_crypto_defs.h b/include/lib/psa/rse_crypto_defs.h
similarity index 100%
rename from include/lib/psa/rss_crypto_defs.h
rename to include/lib/psa/rse_crypto_defs.h
diff --git a/include/lib/psa/rss_platform_api.h b/include/lib/psa/rse_platform_api.h
similarity index 100%
rename from include/lib/psa/rss_platform_api.h
rename to include/lib/psa/rse_platform_api.h
diff --git a/lib/psa/rss_platform.c b/lib/psa/rse_platform.c
similarity index 100%
rename from lib/psa/rss_platform.c
rename to lib/psa/rse_platform.c
-- 
2.53.0

